src/components/checkbox/CheckBox.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import "styles/Checkbox.scss";
```

Comment:
Please include `Checkbox.scss` in` index.scss `as well

Reply:
Done